### DISCUSSION TITLE
Prefetch JSONs are being called on every hover event in v12.2

### DISCUSSION THREAD

**Discussion by sainisagar310** [vercel/next.js#38919] [Help]
In version 12.2, I noticed that when we hover on a `<Link />`, a JSON file and JavaScript chunks gets downloaded. Now if we hover on the same link again then, only the JSON gets downloaded. No matter how many times we're hovering over a link it always download the JSON.

I am not quite sure whether it is expected or an issue. But in the previous version of Next.js it wasn't happening. I think calling the same JSON and hitting the server so many time does not make sense.

https://user-images.githubusercontent.com/26157513/180446083-70abe152-8c85-4b90-8865-ac707992678a.mp4

**Comment by ribeaud** [comment]
Same here with the latest version: _12.3.1_.

**Reply by ribeaud** [reply]
Seems to relate to https://github.com/vercel/next.js/issues/38413 and https://github.com/vercel/next.js/issues/38360. And, up to the posts, this is a feature. Ah, OK... really?
